### DISCUSSION TITLE
Hello

### DISCUSSION THREAD

**Discussion by Indu9597** [crmne/cluster-headache-tracker#82] [Show and tell]
Hello Carmine,
This is Indu. I was the designer that messaged you on Reddit. I am new to github so I am still learning how to log issues or update you. I will be starting the design and will keep you updated.
Thanks.

**Comment by crmne** [comment]
Hi @Indu9597!

Great to see you here! The best way to continue the discussion is to do it in the specific issue I made https://github.com/crmne/cluster-headache-tracker/issues/74.

Discussions are for things that are not connected to specific tasks or issues, so I'll go ahead and close this discussion, but I'll see you on that issue! :)
